test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Rendering/HtmlHelperTextBoxAreaTest.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc.TestCommon;
+using System.ComponentModel.DataAnnotations;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Mvc.Rendering
+{
+    public class HtmlHelperTextBoxAreaTest
+    {
+        [Fact]
+        public void TextAreaFor_GeneratesPlaceholderAttribute_WhenDisplayAttributePromptIsSetAndTypeIsValid()
+        {
+            // Arrange            
+            var model = new TextAreaModelWithAPlaceholder();
+            var helper = DefaultTemplatesUtilities.GetHtmlHelper(model);
+
+            // Act
+            var result = HtmlContentUtilities.HtmlContentToString(helper.TextAreaFor(m => m.Property1));
```

Comment:
Not major i.e. mostly for future reference: Better to break up expressions to make it clear what the real `// Act` is. Here that would mean `var result = helper.TextAreaFor(m => m.Property1);` and doing the `HtmlContentToString()` conversion under `// Assert`.